### DISCUSSION TITLE
Historical Observations by Station and Date

### DISCUSSION THREAD

**Discussion by LatitudeAdjustment** [weather-gov/api#275] [Q&A]
Is it possible to obtain historical observations by station and date?  I am using "/stations/KHPN/observations" to request station KHPN current observations.  As expected a FeatureCollection of observations from today (2020-10-14T12:56:00+00:00) back to 2020-09-19T02:56:00+00:00 is returned.

However, I would like to obtain historical observations by date if possible, something like: "/stations/KHPN/observations/2020-10-01".  Is this data available?  Is there a time format for "/stations/KHPN/observations/{time}" that would allow me to make such a request?

Thanks and best regards,
Mark

**Comment by StephenClouse** [comment]
No, the API does not retain historical data. Our current retention target is 7 days.

Archive data can be requested from [NCEI](https://www.ncei.noaa.gov/).

**Comment by LatitudeAdjustment** [comment]
Ok, API by date would be useful.  I will consult NCEI for archive data.
Could you please explain the use of time parameter and correct format for this API call:
"/stations/KHPN/observations/{time}"?

**Comment by StephenClouse** [comment]
The time parameter should be in ISO8601 format, e.g.:

https://api.weather.gov/stations/KHPN/observations/2020-10-14T14:56:00+00:00

This is for retrieving single observations.

**Comment by LatitudeAdjustment** [comment]
Ok, thank you.
